docs/02-get-started/05-observability.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Observability
+---
+
+We successfully got our Function [triggered](04-trigger-workload-with-event.md).
+Now we would like to check its logs. 
+To do that, we'll use the Grafana dashboard that comes with Kyma. 
+
+> **NOTE:** See how to access logs from the Function's Pod [via Kyma Dashboard](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-in-kyma-dashboard) and [using kubectl](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-using-kubectl). 
+
+1. To access Grafana, forward a local port to a port on the service's Pod:
+    ```bash
+    kubectl -n kyma-system port-forward svc/monitoring-grafana 3000:80
+    ```
+2. In your browser, go to [`http://localhost:3000`](http://localhost:3000) to open Grafana dashboard.
+3. Using the left menu, navigate to **Explore** and choose **Loki** from the dropdown list.
+4. Click on **Log browser** and select the following values:
+   - **1. Select labels to search in**: `container`, `function`
+   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function` 
```

Comment:
I'd reverse the order so it's consistent with the previous step
```suggestion
   - **2. Find values for the selected labels**: for **container** choose `function`, for **function** choose `lastorder`
```

Reply:
Actually, they are given in the order for a reason. This is because there are many containers but not many functions, and if you wanted to first select a container, you'd need to scroll a lot, but if you first select a function, you have the selection of containers narrowed down and you can select it right away, so I'd leave it for convenience.

Reply:
Actually, they are given in the order for a reason. This is because there are many containers but not many functions, and if you wanted to first select a container, you'd need to scroll a lot, but if you first select a function, you have the selection of containers narrowed down and you can select it right away.
See here: 
![image](https://user-images.githubusercontent.com/53601578/133630357-ce60f2c4-4d92-4b9c-af29-9f309b72ef6c.png)
![image](https://user-images.githubusercontent.com/53601578/133630411-d3d4ba7c-a6c5-4ac6-9550-d4e07ab291e3.png)


I'd like to leave it for convenience.

Reply:
Makes total sense!
I guess the UI is also the reason why it is the other way round in line 18?

Reply:
Yes. As you see in the screenshot, that's the order they appear here
![image](https://user-images.githubusercontent.com/53601578/133636318-4f60bbe0-eb95-46a7-852a-4eb110b82d9c.png)